docs/topics/circuit-breaker-mission-basic-interaction-spring-boot-tomcat.adoc

```diff
@@ -16,8 +16,8 @@ Exposes the following endpoints:
 
 * the `/api/greeting` endpoint that you can call to get a personalized greeting response.
 +
-Calling the `/api/greeting` endpoint also calls the `/api/name` endpoint of the `{app-name}-name` service.
-This call happens behind the scenes and is protected by the Circuit Breaker.
+When you call the `/api/greeting` endpoint, it issues a call against the `/api/name` endpoint of the `{app-name}-name` service as part of processing your request.
+Calls made against the `/api/greeting` endpoint are protected by the Circuit Breaker.
```

Comment:
It's actually the calls to the /api/name endpoint that are protected by the CB. Otherwise LGTM.